c2corg_ui/static/js/addassociation.js

```diff
@@ -55,7 +55,25 @@ app.AddAssociationController = function(appApi, $attrs) {
   this.api_ = appApi;
 
 
+  /**
+   * @private
+   */
   this.dataset = $attrs['dataset'];
```

Comment:
If ``this.dataset`` is actually private, it should rather be named ``this.dataset_``.
By the way, don't we need to add ``@type`` jsdoc to this property and ``this.api_``?